pages/agents/agentsfun.jsx

```diff
@@ -133,50 +136,44 @@ const funAgents = [
     title: 'Agent Base',
     anchor: 'agent-base',
     description:
-      'Run your AI influencer on Base. Agent Base offers all the features you love — tweeting, evolving, benefits from meme coins, and interacting autonomously — all powered by the Olas. Get started today & create AI influencer on Base.',
+      'Run your AI influencer on Base. Agent Base offers all the features you love — tweeting, evolving, benefits from meme coins, and interacting autonomously — all powered by the Olas.',
     imgSrc: 'agentsfun-base.png',
-    link: '/pearl#download',
   },
-  // {
```

Comment:
let's keep it commented out, even though we have the config in Pearl, we never pushed it live, so no need to show it as under construction